spec/jwk/ec_spec.rb

```diff
@@ -18,7 +18,7 @@
       let(:keypair) { ec_key.tap { |x| x.private_key = nil } }
       it 'creates an instance of the class' do
         expect(subject).to be_a described_class
-        expect(subject.private?).to eq false
+        expect(subject.private?).to eq true
```

Comment:
This is related to a change in the OpenSSL lib. Think the test is fixed in the current master branch.

Reply:
Hey, I did update my branch with the latest commits on master and this test still kept failing. It fails when I run tests on my local machine on the master branch as well. I'm using ruby-2.4.7 and ruby-3.0.1 for running the tests and rubocop.

Reply:
You're right. I think i fixed it somewhere but apparently not. You'll just have to ignore the failing test for now, sorry for the inconvenience

Reply:
Alright, reverted that spec change! 👍  Anything else pending for this to get merged? Would you like me to add a changelog & minor version(?) update as well?

Reply:
The changelog and versions are usually updated when a new release is cut. Also the fix for the test is now in #427. Did not dig that deep into this but it seems to be a macOS issue

Reply:
Oh okay.